apps/bfd-server/bfd-server-war/pom.xml

```diff
@@ -209,6 +214,21 @@
                 </exclusion>
             </exclusions>
         </dependency>
+		<dependency>
+			<groupId>org.mockito</groupId>
+			<artifactId>mockito-core</artifactId>
+			<version>3.10.0</version>
+			<scope>test</scope>
+		</dependency>
+		<!-- Hibernate brings in an older version of byte-buddy which seems
+		to conflict with mockito, which wants version 1.11.0.  Bringing it
+		in as a first class dependency to resolve the issue. -->
+		<dependency>
+			<groupId>net.bytebuddy</groupId>
```

Comment:
Do we need a Sia for this?

Reply:
ByteBuddy is already brought in by both hibernate and Mockito as a transitive dependency, this just brings it in as first class so that Mockito gets the right version of it (and only for testing scope).  That being said, I defer to others about if we should go into further analysis of it.